packages/react-native-reanimated/src/jestUtils.ts

```diff
@@ -27,22 +27,81 @@ const defaultFramerateConfig = {
   fps: 60,
 };
 
+const isEmpty = (obj: object) => Object.keys(obj).length === 0;
+const getStylesFromObject = (obj: object) => {
+  return obj === undefined
+    ? {}
+    : Object.fromEntries(
+        Object.entries(obj).map(([property, value]) => [
+          property,
+          value.value ?? value,
+        ])
+      );
+};
+
+type StyleValue = { value: unknown };
+type JestInlineStylesType =
+  | {
+      [s: string]: StyleValue;
+    }
+  | ArrayLike<StyleValue>;
```

Comment:
```suggestion
type JestInlineStyle =
  | {
      [s: string]: StyleValue;
    }
  | ArrayLike<StyleValue>;
```

Reply:
Changed in [19bae71](https://github.com/software-mansion/react-native-reanimated/pull/6400/commits/19bae71cfcf30605d8f720542886999ae8108803)